packages/grid/test/keyboard-navigation-cell-button.test.js

```diff
@@ -0,0 +1,93 @@
+import { expect } from '@esm-bundle/chai';
+import { arrowLeft, arrowRight, aTimeout, fixtureSync } from '@vaadin/testing-helpers';
+import sinon from 'sinon';
+import '../vaadin-grid.js';
+import { flushGrid } from './helpers.js';
+
+let grid;
+
+function getRowCell(rowIndex, cellIndex) {
+  return grid.$.items.children[rowIndex].children[cellIndex];
+}
+
+function getRowFirstCell(rowIndex) {
+  return getRowCell(rowIndex, 0);
+}
+
+describe('keyboard navigation - cell button', () => {
```

Comment:
```suggestion
describe('keyboard navigation - focus button mode', () => {
```

Can also rename the test file.